app/views/quick_search/search/index.html.erb

```diff
@@ -49,7 +48,14 @@
                                   data-quicksearch-page=""
                                   data-quicksearch-template=""></p>
         </div>
-      <% end %>  
+      <% end %>
```

Comment:
Is this already on Lines 46 - 50? Conditionally rendered in an else?

Reply:
Oh huh. Yes, that's right. Sorry, I'm getting all mixed up today.